### PR TITLE
ins update

### PR DESCRIPTION
**Pull request by mcamurri**
This new feature allows the INS module tu be set with a fixed bias state and never changed over time by the filter.
The feature is retro-compatible with previous code. To enable it, new params need to be defined in the config file, for example:
```
    accel_bias_update = true;
    accel_bias_initial = [0.0001, 0.0002, 0.0003];

    gyro_bias_update = false;
    gyro_bias_initial = [0.0001, 0.0002, 0.0003];

    online_bias_initial = false;
```
When set to true, `accel_bias_update` and `gyro_bias_update` will let the EKF estimate the bias.
The initial values for the bias can be set through `accel_bias_initial` and `gyro_bias_initial`.
The value `online_bias_initial` allows for ignoring the initial bias initialization, in case you want to use a precomputed value.

The example above ignores the initial online estimation, and computes online just the accel bias, starting with the indicated initial values.

Note that this merge request starts from the [latest commit ponted by pronto-distro](https://github.com/openhumanoids/pronto/tree/bbcce4a16692592f45e78d32065cb6f5e8d8a0fa), from which I had to add few modifications to add a new sensor to the filter, (see [here](https://github.com/openhumanoids/pronto/compare/master...iit-DLSLab:dls-devel)).

Therefore, the pull request is not mergeable.

### CODE REVIEW

File: mav/mav-state-estimator/src/mav_state_est/rbis_update_interface.cpp [hunk 1]
```diff
@@ -2,9 +2,9 @@
 
 namespace MavStateEst {
 
-const char * RBISUpdateInterface::sensor_enum_chars = "igvlfsorxdukepabmtw";
+const char * RBISUpdateInterface::sensor_enum_chars = "igvlfsorxdukepabmtwh";
 const char * RBISUpdateInterface::sensor_enum_strings[] =
-    { "ins", "gps", "vicon", "laser", "laser_gpf", "scan_matcher", "optic_flow", "reset", "invalid", "rgbd", "fovis", "legodo", "legodo_external", "pose_meas", "altimeter", "airspeed", "sideslip", "init_message", "viewer" };
+    { "ins", "gps", "vicon", "laser", "laser_gpf", "scan_matcher", "optic_flow", "reset", "invalid", "rgbd", "fovis", "legodo", "legodo_external", "pose_meas", "altimeter", "airspeed", "sideslip", "init_message", "viewer", "hyq_kse" };
```
Comment (mauricefallon) [inline mav/mav-state-estimator/src/mav_state_est/rbis_update_interface.cpp]: please revert

File: mav/mav-state-estimator/src/mav_state_est/mav_state_est.cpp [hunk 2]
```diff
@@ -27,40 +27,57 @@ MavStateEstimator::~MavStateEstimator()
 
 void MavStateEstimator::addUpdate(RBISUpdateInterface * update, bool roll_forward)
 {
+    // Add current update to history
```
Comment (mauricefallon) [inline mav/mav-state-estimator/src/mav_state_est/mav_state_est.cpp]: thanks for the comments, can you put into a separate PR

File: mav/mav-state-estimator/src/mav_state_est/rbis_update_interface.hpp [hunk 3]
```diff
@@ -8,7 +8,7 @@ namespace MavStateEst {
 class RBISUpdateInterface {
 public:
   typedef enum {
-    ins, gps, vicon, laser, laser_gpf, scan_matcher, optical_flow, reset, invalid, rgbd, fovis, legodo, legodo_external, pose_meas, altimeter, airspeed, sideslip, init_message, viewer
+    ins, gps, vicon, laser, laser_gpf, scan_matcher, optical_flow, reset, invalid, rgbd, fovis, legodo, legodo_external, pose_meas, altimeter, airspeed, sideslip, init_message, viewer, hyq_kse
```
Comment (mauricefallon) [inline mav/mav-state-estimator/src/mav_state_est/rbis_update_interface.hpp]: zap

File: mav/mav-state-estimator/src/mav_state_est/sensor_handlers.cpp [hunk 4]
```diff
@@ -46,6 +50,42 @@ InsHandler::InsHandler(BotParam * _param, BotFrames * _frames)
   g_vec_sum.setZero();
   mag_vec_sum.setZero();
   gyro_bias_sum.setZero();
+    int tmp_int;
```
Comment (mauricefallon) [inline mav/mav-state-estimator/src/mav_state_est/sensor_handlers.cpp]: can you make the new parameters required. While the param system supports parsing them making all parameters mandatory makes config errors easier to spot (because the application asserts if it can't find them)

**Comment by mcamurri** [comment]
As requested, I've made two separated pull request, one with ins and one with comments. I've also made a third one for having the vicon in yaw and orientation modes.
